src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -50,6 +50,25 @@ export default class SimidMapCreative extends BaseSimidCreative {
    * @private 
   */
   grantLocationAccess_() {
-    //ToDo(kristenmason@): implement map
+    this.loadMap_();
+  }
+
+  /**
+   * Loads a map object that currently displays a hardcoded location.
+   * @param {Object} coordinates - the LatLng object of user's current location
+   * ToDo(kristenmason@): implement grant location access and modify
+   * function to pass in current position (currently coords default to GooglePlex)
+   * @private 
+  */
+  loadMap_(coordinates = new google.maps.LatLng(37.422004,-122.081402)) { 
+    const map = new google.maps.Map(document.getElementById('map'), {
+      zoom: 13,
```

Comment:
Make const

Reply:
Are you talking about zoom here? I left it as is because in the Maps API it is one of the [MapOptions properties](https://developers.google.com/maps/documentation/javascript/reference/map#MapOptions). If it's a property should I still use const here?

Reply:
I meant the 13. The zoom can def stay as is, but 13 can be a const maybe `DEFAULT_ZOOM` or `INITIAL_ZOOM` or something like that